src/main/flatbuffers/rlbot.fbs

```diff
@@ -0,0 +1,188 @@
+
+namespace rlbot.flat;
+
+table ControllerState {
+  /// -1 for full reverse, 1 for full forward
+  throttle:float;
+
+  /// -1 for full left, 1 for full right
+  steer:float;
+
+  /// -1 for nose down, 1 for nose up
+  pitch:float;
+
+  /// -1 for full left, 1 for full right
+  yaw:float;
+
+  /// -1 for roll left, 1 for roll right
+  roll:float;
+
+  /// true if you want to hold the jump button
+  jump:bool;
+
+  /// true if you want to hold the boost button
+  boost:bool;
+
+  /// true if you want to hold the handbrake button
+  handbrake:bool;
+}
+
+table PlayerInput {
+  playerIndex:int;
+  controllerState:ControllerState;
+}
+
+root_type PlayerInput;
+
+// Values are in "unreal units"
+struct Vector3 {
+  x:float;
+  y:float;
+  z:float;
+}
+
+// Values are in radians
+struct Rotator {
+  pitch:float;
+  yaw:float;
+  roll:float;
+}
+
+table Touch {
+  /// The name of the player involved with the touch.
+  playerName:string;
+
+  /// Seconds that had elapsed in the game when the touch occurred.
+  gameSeconds:float;
+
+  /// The point of contact for the touch.
+  location:Vector3;
+
+  /// The direction of the touch.
+  normal:Vector3;
+}
+
+table ScoreInfo {
+  score:int;
+  goals:int;
+  ownGoals:int;
+  assists:int;
+  saves:int;
+  shots:int;
+  demolitions:int;
+}
+
+table Physics {
+  location:Vector3;
+  rotation:Rotator;
+  velocity:Vector3;
+  angularVelocity:Vector3;
+}
+
+table PlayerInfo {
+  physics:Physics;
+  scoreInfo:ScoreInfo;
+  isDemolished:bool;
+  hasWheelContact:bool;
+  isSupersonic:bool;
+  isBot:bool;
+  jumped:bool;
+  doubleJumped:bool;
+  name:string;
+  team:int;
+  boost:int;
+}
+
+table BallInfo {
+  physics:Physics;
+  latestTouch:Touch;
+}
+
+table BoostPadState {
+  /// True if the boost can be picked up
+  isActive:bool;
+
+  /// The number of seconds since the boost has been picked up, or 0.0 if the boost is active.
+  timer:float;
+}
+
+table GameInfo {
+  secondsElapsed:float;
+  gameTimeRemaining:float;
+  isOvertime:bool;
+  isUnlimitedTime:bool;
+  isRoundActive:bool;
+  isKickoffPause:bool;
+  isMatchEnded:bool;
+}
+
+table GameTickPacket {
+  players:[PlayerInfo];
+  boostPadStates:[BoostPadState];
+  ball:BallInfo;
+  gameInfo:GameInfo;
+}
+
+root_type GameTickPacket;
+
+// This section deals with arena information, e.g. where the goals and boost locations are.
```

Comment:
why are all other comments /// and this one is just //   ?

Reply:
/// comments are translated into documentation in the target language, e.g. javadoc in java. This one is just a // because I only intend for it to be seen by people looking at this file.